catalogue/webapp/services/catalogue/images.js

```diff
@@ -0,0 +1,43 @@
+import fetch from 'isomorphic-unfetch';
+import type {
+  CatalogueApiError,
+  CatalogueResultsList,
+  Image,
+} from '@weco/common/model/catalogue';
+import { type CatalogueImagesApiProps } from '@weco/common/services/catalogue/api';
+import { serialiseUrl } from '@weco/common/services/catalogue/routes';
+import { type Environment, rootUris } from './common';
+
+type GetImagesProps = {|
+  params: CatalogueImagesApiProps,
+  pageSize?: number,
+  ...Environment,
+|};
+
+export async function getImages({
+  params,
+  env = 'stage', // TODO switch this to prod once the images search is deployed
```

Comment:
We are having some issues with the pipeline that is delaying deploying the images search stuff to the prod API. So as not to delay this, and as it's behind a toggle, I thought I'd put this here for now.